### PR TITLE
updates dismiss icon and reflows text around cta button

### PR DESCRIPTION
**Pull request by pixelandpage**
Updates the dismiss icon to the `x` in a circle and reflows the paragraph text around cta button - some spacing changes to accommodate these.

### CODE REVIEW

File: myft-digest-promo/main.scss [hunk 1]
```diff
@@ -85,10 +94,14 @@ $icon-width-m: 90px;
...
 	display: flex;
+	overflow: auto;
 	align-items: flex-end;
 	justify-content: flex-end;
-	padding-bottom: 12px;
+	padding-bottom: 15px;
 	clear: both;
+	@include oGridRespondTo('M') {
+			padding-left: 2px;
+		}
```
Comment (phamann) [inline myft-digest-promo/main.scss]: indentation

File: myft-digest-promo/main.scss [hunk 2]
```diff
@@ -17,26 +17,34 @@ $icon-width-m: 90px;
...
+}
+
+.n-myft-digest-promo__cta-wrapper {
+	padding-left: 2px;
 }
 
 .n-myft-digest-promo-dismiss-btn__container {
 		@include oGridRespondTo('M') {
-		@include oGridColspan($span: 7);
+		@include oGridColspan($span: 7.5);
```
Comment (phamann) [inline myft-digest-promo/main.scss]: Does this work? Surely we should be using a multiple of one here.

**Comment by phamann** [comment]
Looks like you have merge conflicts.

**Comment by phamann** [comment]
👍 other than comments

File: myft-digest-promo/main.scss (hunk 2 above)
Comment (pixelandpage) [inline myft-digest-promo/main.scss]: It works, but apparently it shouldn't so we're making this `$span: 8`

**Pull request merged**